packages/manager/apps/web/client/app/domain/webhosting/order/steps/duration/domain-webhosting-order-steps-duration.controller.js

```diff
@@ -20,11 +20,5 @@ export default class {
 
   updateDuration() {
```

Comment:
This function does not seem to be called anymore, maybe a cleanup is in order

Reply:
It is called

![image](https://github.com/ovh/manager/assets/8767496/d9a6293a-5824-4463-a9b4-23771639fb6a)
![image](https://github.com/ovh/manager/assets/8767496/e92825bf-38b0-45f2-b4f0-5306c7d864ca)